recipes/librdkafka/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
+export CPPFLAGS="$CPPFLAGS -I$PREFIX/include"
+export C_INCLUDE_PATH="$PREFIX/include"
+export LIBRARY_PATH="$PREFIX/lib"
+
+./configure --prefix=$PREFIX
+make -j 2
```

Comment:
That guy, listed at the top of the maintainers list, is a lazy bum that forgot to update that recipe :stuck_out_tongue_winking_eye: